api/valor_api/backend/core/label.py

```diff
@@ -229,38 +197,40 @@ def create_labels(
         return []
 
     # remove duplicates
-    labels_no_duplicates = list(set(labels))
-
-    # get existing labels
-    existing_labels = {
-        (label.key, label.value)
-        for label in fetch_matching_labels(db, labels_no_duplicates)
-    }
-
-    # create new labels
-    new_labels_set = {
-        (label.key, label.value)
-        for label in labels_no_duplicates
-        if (label.key, label.value) not in existing_labels
-    }
-    new_labels = [
-        models.Label(key=label[0], value=label[1])
-        for label in list(new_labels_set)
+    values = [
+        {"key": label.key, "value": label.value} for label in set(labels)
     ]
+    insert_stmt = (
```

Comment:
Ive tried to replicate using asyncio but have had no success. I have asked the thread that raised this issue if anyone else is able to reproduce.

If it behaves like previous label creation bugs, it will be rare and intermittent.